### DISCUSSION TITLE
Ignore options usage

### DISCUSSION THREAD

**Discussion by m0t0rh3ad** [andresch/trackfs#33] [Q&A]
Hello, I got confused how to properly use --ignore-tags option. Can someone explain, how should it be configured to ignore Artist and Album in filename? Thanks

**Comment by andresch** [comment]
Not sure I fully understand what you try to achieve, but the --ignore-tags option has nothing to do with the filename. It just defines, which of the meta-data tags that are included flac+cue file will *NOT* be included as tags in the metadata of the individual track files.

**Reply by m0t0rh3ad** [reply]
let me try to explain
right now I've got next in the output folder:
/Artist/Year -Album (Release-Date) (Label-and-Number)/Artist -Album.flac.#-#.Track-Number.Album.flac
example:
/Ayreon/1996\ -\ Actual\ Fantasy\ \(1996-11-07\)\ \(Victor\ VICP-5798\)/Ayreon\ -\ Actual\ Fantasy.flac.#-#.001.Actual\ Fantasy.flac
Is it possible to get output like this:
/Artist/Year - Album/Track-Number - Track-Name.flac?

**Reply by andresch** [reply]
Unfortunately trackfs has no such option. 

trackfs has no capability to transform the structure of the source directly tree. It simply creates a virtual clone of the source directory, where all it does is to replace individual album files, with a sequence of individual track files. Unfortunately (as I just explained in #33 ) also names of the individual track files can't really get adjusted.

**Reply by m0t0rh3ad** [reply]
Thanks for the explanations and trackfs itself.
